android-components/samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt

```diff
@@ -170,7 +173,18 @@ open class DefaultComponents(private val applicationContext: Context) {
                 PromptMiddleware(),
                 SessionPrioritizationMiddleware(),
             ) + EngineMiddleware.create(engine),
-        )
+        ).apply {
+            WebNotificationFeature(
+                applicationContext,
+                engine,
+                icons,
+                R.mipmap.ic_launcher_foreground,
```

Comment:
Used this to avoid adding a resource and because the full launcher icon was barely visible.